admin/src/pages/Products/index.jsx

```diff
@@ -8,22 +8,39 @@ import {
   updateProductsRequest,
   addProductsRequest,
   deleteProductRequest,
+  getTopSalesRequest,
+  getPopTagRequest,
 } from "../../actions";
 import DataTable from "./components/DataTable";
 import NewModal from "../../components/UI/Modal";
+import TopSalesChart from "./components/Chart/TopSalesChart";
+import PopTagsChart from "./components/Chart/PopTagsChart";
 import axios from "../../helper/axios";
 import { BsXSquare, BsSearch } from "react-icons/bs";
 import "./style.scss";
+import "antd/dist/antd.css";
+import { Select } from "antd";
+import { Spinner } from "../../components/UI/Spinner";
+import { useTranslation } from "react-i18next";
+
+const { Option } = Select;
 
 /**
  * @author
  * @function Home
  **/
 
+const thisMonth = new Date().getMonth() + 1;
```

Comment:
```suggestion
const THIS_MONTH = new Date().getMonth() + 1;
```
quăng vào constant luôn anh